discovery-provider/src/queries/get_health.py

```diff
@@ -107,8 +106,15 @@ def _get_query_insights():
 
 def _get_chain_health():
     try:
-        response = requests.get(CHAIN_HEALTH_ENDPOINT, timeout=0.02)
-        return response.json()
+        health_res = requests.get(LOCAL_RPC + '/health')
+        chain_res = health_res.json()
+
+        web3 = get_nethermind_web3()
+        latest_block = web3.eth.get_block('latest')
+        chain_res['block_number'] = latest_block.number
+        chain_res['hash'] = latest_block.hash.hex
+        chain_res['chain_id'] = web3.eth.chain_id
+        return chain_res
```

Comment:
Would it be a good idea to call this admin rpc as well? We could gather the network ID the node think it's on along with some other info. https://docs.nethermind.io/nethermind/ethereum-client/json-rpc/admin#admin_nodeinfo

Reply:
There's also a debug module we could turn on and off where we could maybe get the same info. https://docs.nethermind.io/nethermind/ethereum-client/json-rpc/debug#debug_getconfigvalue

Reply:
This method could also be useful because we can see the "recents". Meaning those signers that are not eligible to sign new blocks. https://docs.nethermind.io/nethermind/ethereum-client/json-rpc/clique#clique_getsnapshot

Reply:
yea good call out we'll need to add the admin module here first https://github.com/AudiusProject/audius-docker-compose/blob/main/discovery-provider/chain/config.cfg#L24. i'll follow up w you on that. i'm a little hesitant there might be sensitive info in there. 

i'll make a change to expose the spec.json that gets generated via audius-cli and that should be enough info https://github.com/AudiusProject/audius-docker-compose/blob/main/audius-cli#L730

Reply:
> i'm a little hesitant there might be sensitive info in there.
Definitely. That debug endpoint may be able to give us similar info without admin control. If we're already getting the chain id though maybe we don't need it today.